mobilecoind/clients/python/mob_client/mob_client.py

```diff
@@ -163,7 +164,7 @@ def read_transfer_code(self, b58_code):
         """
         request = api.ReadTransferCodeRequest(b58_code=b58_code)
         response = self.stub.ReadTransferCode(request)
-        return response.entropy, response.tx_public_key, response.memo
+        return response.entropy, response.tx_public_key, response.memo, response.utxo
```

Comment:
I think it would be better to just return `response`. This tuple means every change to it is a breaking change.